File: app/src/main/java/arunkbabu90/popmovies/network/MovieResponse.java
package arunkbabu90.popmovies.network;


import com.google.gson.annotations.SerializedName;

import java.util.List;

import arunkbabu90.popmovies.database.MoviePopular;

public class MovieResponse
{
    @SerializedName("results") private List<MoviePopular> results;

    public List<MoviePopular> getPopularResults() {
        return results;
    }
}

File: app/src/main/java/arunkbabu90/popmovies/data/model/MovieResponse.kt
package arunkbabu90.popmovies.data.model

import com.google.gson.annotations.SerializedName

data class MovieResponse(val page: Int,
                         @SerializedName("results") val movies: List<Movie>,
                         @SerializedName("total_pages") val totalPages: Int,
                         @SerializedName("total_results") val totalMovies: Int)